data/mil2525d/utilities/style-utilities/image-conversion-utilities/README.md

```diff
@@ -39,3 +39,20 @@
     *  Check the output for errors
 *  Check the destination folder for the converted image files
     * IMPORTANT NOTE: MS Paint (which is the default file association) does not support the latest version of EMF so you will need to use an alternative program to view the resulting EMF files (ex. MS Word)
+
+## Alternative Workflow using Adobe Illustrator Actions
+
+* If you have access to Adobe Illustrator (We are using CS6), you can use Actions in Batch Mode to convert your files from .svg to .emf.
+* IMPORTANT: This process will export .emfs to one singular folder, not the origin .svg file directory.
+* To start, download the `BatchConvertSVGtoEMF.aia` action set file contained in this repository.
+* Open Adobe Illustrator, navigate to the Window pane, and open up Actions.
+* Choose Load Actions from the Actions Panel menu. Navigate to the ExportasEMF action set, and click Open. 
+* Under the `ExportasEMF` action, double click on the destination file path and edit it to ensure that your files will be converted to .emf in the desired location.
+* Click on the `ExportasEMF action` within the action set. Then, navigate to the Actions Panel menu on the top right of the pane and choose Batch. This will open up the Batch pane.
+    * Set the source folder to where your .svg files are located.
+    * Make sure that the `Include All Subdirectories` check box is checked if your .svg files are stored in subdirectories.
+    * Set the destination to `Save and Close.` You can specify the output folder by choosing `Override Action 'Export' Commands.`
```

Comment:
"You can specify output" or "You *must* specify output" - do I need to do this or not?